src/intl/en/page-about.json

```diff
@@ -24,14 +24,13 @@
   "page-about-p-8": "Do you have an idea for how to improve ethereum.org? We'd love to collaborate with you!",
   "page-what-is-ethereum-energy-consumption-chart-legend": "Annual Energy Consumption in TWh/yr",
   "page-upgrades-post-merge-banner-governance-ood": "Some content on this page is out-of-date after the merge. Please raise a PR if you would like to contribute.",
-  "energy-consumption-chart-youtube-label": "YouTube",
-  "energy-consumption-chart-gold-mining-galaxy-label": "Gold mining (Galaxy Digital)",
   "energy-consumption-chart-global-data-centers-label": "Global data centers",
-  "energy-consumption-chart-gold-mining-cbeci-label": "Gold mining (CBECI)",
+  "energy-consumption-gold-mining-cbeci-label": "Gold mining",
   "energy-consumption-chart-btc-pow-label": "BTC PoW",
   "energy-consumption-chart-netflix-label": "Netflix",
   "energy-consumption-chart-eth-pow-label": "ETH PoW",
   "energy-consumption-chart-gaming-us-label": "Gaming in the US",
   "energy-consumption-chart-paypal-label": "PayPal",
+  "energy-consumption-chart-airbnb-label": "AirBnB",
```

Comment:
Aside: any point in translating these?

cc: @lukassim

Reply:
Probably not, 'Gaming in the US' is the only one that would get translated in the vast majority of languages. There's also some languages that write names like PayPal & AirBnB in their native script, but in most languages names are still written using Latin characters

Reply:
should we should remove these labels then?

- "energy-consumption-chart-btc-pow-label"
- "energy-consumption-chart-netflix-label"
- "energy-consumption-chart-eth-pow-label"
- "energy-consumption-chart-paypal-label"
- "energy-consumption-chart-airbnb-label"

cc @lukassim 

I'll open a PR if it's ok